frontend/src/stores/useOptionsFormDataStore/index.ts

```diff
@@ -0,0 +1,179 @@
+            const match = re.exec(elName)
+            if (match) {
+                highest = Math.max(highest, Number(match[1]))
+            }
+        }
+        return highest
+    }
+
+    return {
+        asyncStatus: computed(() =>
+            definitionAsyncStatus.value === 'idle' && dataAsyncStatus.value === 'idle' ? 'idle' : 'loading',
+        ),
+        error: computed(() => definitionState.value.error ?? dataState.value.error ?? mutationState.value.error),
+        isSaving: computed(() => postDataAsyncStatus.value !== 'idle'),
+        isClean: computed(() => areFormDataObjIdentical(formData.value, formDataClean.value)),
+
+        formDefinition: computed(() => definitionState.value.data),
+        formData,
+        formErrors,
+
+        /**
+         * Submits the form data.
+         *
+         * @returns `false` if form has validation errors, otherwise `true`.
+         */
+        async submit(): Promise<boolean> {
+            try {
+                const rawFormData = toRaw(formData.value)
+                formErrors.value = await postData(rawFormData)
+                formDataClean.value = structuredClone(rawFormData)
+                mutationState.value.error = null
+            } catch (err) {
+                if (err instanceof Error) {
+                    mutationState.value.error = err
+                }
+                throw err
+            }
+            return Object.keys(formErrors.value).length === 0
+        },
+
+        /** Resets the form data. */
+        reset(): void {
+            formData.value = structuredClone(toRaw(formDataClean.value))
+        },
+
+        /**
+         * Retrieves a value from the form data object.
+         *
+         * @param name The name representing the input field, e.g. `general[first_name]`.
+         * @returns The value found at the specified name in the `formData` object, or `undefined` otherwise.
+         */
+        getValue<T extends OptionsFormDataValue>(name: string): T | undefined {
+            if (name in formData.value) {
+                return formData.value[name] as T
+            }
+        },
+
+        /**
+         * Sets a value on the form data object.
+         *
+         * @param name The name representing the input field, e.g. `general[first_name]`.
+         * @param value The value to set at the specified name.
+         */
+        setValue(name: string, value: OptionsFormDataValue): void {
+            formData.value[name] = value
+        },
+
+        /**
+         * Retrieves the validation feedback text of a form element.
+         *
+         * @param path The path representing the element.
+         * @returns The validation feedback text or `undefined`.
+         */
+        getFeedback(path: string[]): string | undefined {
+            if (path[0] === 'general') {
+                path.shift()
+            }
+            const key = path.join('.')
+            return formErrors.value[key]
```

Comment:
Pydantic indiziert Listen (Also hier relevant: Repetitions) von 0, du fängst bei 1 an. Daher werden Fehler in Repetitions ein Item zu weit oben angezeigt.